tools/fiptool/fiptool.c

```diff
@@ -826,7 +826,7 @@ static void create_usage(void)
 	    "pointed to by file.\n");
 	printf("  --plat-toc-flags <value>\t16-bit platform specific flag field "
 	    "occupying bits 32-47 in 64-bit ToC header.\n");
-	fputc('\n', stderr);
+	printf("\n");
```

Comment:
I do not have a strong opinion about whether help messages should go to stdout or stderr, as long as I see consistency.   Whether stdout or stderr is a matter of taste.  For example,  "git help" outputs usage messages to stdout.